nodejs/lesson01/solution/hello.js

```diff
@@ -1,33 +1,24 @@
-'use strict';
-
-const assert = require('assert');
-const initTracer = require('../../lib/tracing').initTracer;
-
-function sayHello(helloTo) {
-    var span = tracer.startSpan('say-hello');
-    span.setTag('hello-to', helloTo);
-
-    var helloStr = `Hello, ${helloTo}!`;
-    span.log({
-        'event': 'string-format',
-        'value': helloStr
-    });
-    
-    console.log(helloStr);
-
-    span.log({'event': 'print-string'})
-    span.finish();
-}  
-  
-assert.ok(process.argv.length == 3, 'expecting one argument');
-
+const assert = require("assert");
+const { initTracer } = require("../../lib/tracing");
+
+const tracer = initTracer("hello-world");
+
+const sayHello = helloTo => {
```

Comment:
why not define this as a proper function?
```javascript
function sayHello(helloTo) {
}
```

Reply:
I'm just writing it as an arrow function, to keep the style consistent throughout.